wasm/src/CoinTypeExtension.d.ts

```diff
@@ -0,0 +1,13 @@
+export class CoinTypeExtension {
```

Comment:
Is it possible to extend `CoinType` using `prototype`?
```ts
CoinType.prototype.derivationPath = function(): string {
    // TODO
}  
```

Reply:
Or it would be more convenient to use static function:
```ts
export class CoinTypeExtension {
    static derivationPath(coinType: CoinType): string
}
```

Reply:
I think it's better to avoid static function and prototype and keep the current approach, if we really want static function then we do not need to have `CoinTypeExtension` we can just extend CoinType:

```cpp
EMSCRIPTEN_BINDINGS(Wasm_TWCoinType) {
    enum_<TWCoinType>("CoinType")
        .value("aeternity", TWCoinType::TWCoinTypeAeternity);
        emscripten::function("derivationPath", &TWCoinTypeDerivationPath);
}
```

The current approach make the usage more convenient by following the extension.

ofc using prototype would be ideal for codegen, but then we need to figure out how the codegen will generate the typescript